src/util.jl

```diff
@@ -0,0 +1,40 @@
+# LibFTD2XX.jl - Utility Module
+#
+# By Reuben Hill 2019, Gowerlabs Ltd, reuben@gowerlabs.co.uk
+#
+# Copyright (c) Gowerlabs Ltd.
+
+module Util
+
+export ntuple2string
+
+using Compat
+
+"""
+    ntuple2string(input::NTuple{N, Cchar} where N)
+
+Convert an NTuple of Cchars (optionally null terminated) to a julia string.
+
+# Example
+
+```jldoctest
+julia> ntuple2string(Cchar.(('h','e','l','l','o')))
+"hello"
+
+julia> ntuple2string(Cchar.(('h','e','l','l','o', '\0', 'x'))) # null terminated
+"hello"
+```
+
+"""
+function ntuple2string(input::NTuple{N, Cchar} where N)
+  if any(input .== 0)
+    @compat endidx = findall(input .== 0)[1]-1
```

Comment:
Compat required for what? 0.6 dropped now, won't flag again - find all instances, and remove Compat dependency if no longer required.

Reply:
Resolved in 76caea9553a6f76f7d912b5b5376938c439eceb7